main.cpp

```diff
@@ -18,16 +18,25 @@ using namespace clang::ast_matchers;
 using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
+private:
+	Rewriter* rewriter_;
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
+	CastCallBack(Rewriter& rewriter) {
+		rewriter_ = &rewriter;

```

Comment:
How to rewrite this code piece to use `member initializer list`?

Reply:
```
private: 
    Rewriter &rewriter_;
public:
    CastCallBack(Rewriter& rewriter): rewriter_(rewriter) {}
```